multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -165,842 +158,159 @@ fn payments_were_adjusted_due_to_insufficient_balances() {
         permanent_debt_allowed_gwei: 10_000_000,
         unban_below_gwei: 1_000_000,
     };
-    let owed_to_serv_node_1_minor =
-        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei + 5_000_000);
-    let owed_to_serv_node_2_minor =
-        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei + 20_000_000);
-    let owed_to_serv_node_3_minor =
-        gwei_to_wei::<u128, _>(payment_thresholds.debt_threshold_gwei + 60_000_000);
     // Assuming all Nodes rely on the same set of payment thresholds
-    let cons_node_initial_service_fee_balance_minor = (owed_to_serv_node_1_minor
-        + owed_to_serv_node_2_minor)
-        - gwei_to_wei::<u128, u64>(2_345_678);
-    let agreed_transaction_fee_unit_price_major = 60;
-    let transaction_fee_needed_to_pay_for_one_payment_major = {
-        // We will need less but this should be accurate enough
-        let txn_data_with_maximized_cost = [0xff; 68];
+    let owed_to_serv_node_1_minor = to_wei(payment_thresholds.debt_threshold_gwei + 5_000_000);
+    let owed_to_serv_node_2_minor = to_wei(payment_thresholds.debt_threshold_gwei + 20_000_000);
+    // Account of Node 3 will be a victim of tx fee insufficiency and will fall away, as its debt
+    // is the heaviest, implying the smallest weight evaluated and the last priority compared to
```

Comment:
In this comment you want `least` instead of `last` (just a typo).

You'd also like to mention that due to insufficiency only 2 payments can be processed on the blockchain.